internal/supervisor/installation.go

```diff
@@ -329,7 +329,7 @@ func (s *InstallationSupervisor) createClusterInstallation(cluster *model.Cluste
 	clusterInstallation := &model.ClusterInstallation{
 		ClusterID:      cluster.ID,
 		InstallationID: installation.ID,
-		Namespace:      model.NewID(),
+		Namespace:      installation.ID,
```

Comment:
:tada:

Reply:
this is just for new installations, right? if run an update in an existing installation it will still work?

Reply:
Yes just new installations, old ones will work fine